ddtrace/appsec/processor.py

```diff
@@ -90,3 +90,7 @@ def on_span_finish(self, span):
             span._set_str_tag("appsec.event", "true")
             span._set_str_tag("_dd.appsec.json", '{"triggers":%s}' % (res,))
             span.set_tag(MANUAL_KEEP_KEY)
+
+    def shutdown(self):
+        # type: () -> None
+        pass
```

Comment:
should we just have the base implementation be `pass` and then base classes can override if they want. or do we want to force everyone to implement this `pass`?

Reply:
ah that's a good idea. Not all processors have to implement it 👍